pkg/registry/core/pod/strategy.go

```diff
@@ -239,6 +239,18 @@ func getPod(getter ResourceGetter, ctx context.Context, name string) (*api.Pod,
 	return pod, nil
 }
 
+// returns primary IP for a Pod
```

Comment:
getPodIP returns the primary IP for a Pod. Returns an empty string if pod is nil or there are no IPs